packages/@lightningjs/ui-components/src/components/FocusManager/FocusManager.stories.js

```diff
@@ -135,3 +74,26 @@ class ButtonFixedWidth extends Button {
     super._init();
   }
 }
+
+ColumnWithRows.args = {
+  direction: 'row',
+  wrapSelected: false
+};
+
+ColumnWithRows.argTypes = {
+  direction: {
+    control: 'radio',
+    options: ['row', 'column'],
+    description: 'The navigation direction for focus (left/right or up/down)',
+    table: {
+      defaultValue: { summary: 'row' }
+    }
+  },
+  wrapSelected: {
+    control: 'boolean',
+    description: 'Enables wrapping behavior for focus navigation',
+    table: {
+      defaultValue: { summary: false }
+    }
+  }
+};
```

Comment:
I'm not seeing these controls at all in the `ColumnWithRows` story 🤔

Reply:
Weird, they are showing up for me and I can interact with them.

Reply:
Yeah, I can see the controls too.
I forgot to add detailed description on wrapSelected control in mdx file, Can you add more detailed explanation something like "When wrapSelected is set to true, the focus will loop back to the beginning of the list after reaching the last item, and vice versa. This enables continuous navigation through the list without dead ends. If wrapSelected is set to false (the default value), the focus will stop at the first or last item, depending on the navigation direction".